Jenkinsfile

```diff
@@ -7,48 +7,31 @@ def propagateParamsToEnv() {
   }
 }
 
-properties([
-  disableConcurrentBuilds(),
-  parameters([
-    choice(choices: ["run", "skip"].join("\n"),
-           defaultValue: 'run',
-           description: 'Run or skip robotest system wide tests.',
-           name: 'RUN_ROBOTEST'),
-    choice(choices: ["true", "false"].join("\n"),
-           defaultValue: 'true',
-           description: 'Destroy all VMs on success.',
-           name: 'DESTROY_ON_SUCCESS'),
-    choice(choices: ["true", "false"].join("\n"),
-           defaultValue: 'true',
-           description: 'Destroy all VMs on failure.',
-           name: 'DESTROY_ON_FAILURE'),
-    choice(choices: ["true", "false"].join("\n"),
-           defaultValue: 'true',
-           description: 'Abort all tests upon first failure.',
-           name: 'FAIL_FAST'),
-    choice(choices: ["gce"].join("\n"),
-           defaultValue: 'gce',
-           description: 'Cloud provider to deploy to.',
-           name: 'DEPLOY_TO'),
-    string(name: 'PARALLEL_TESTS',
-           defaultValue: '4',
-           description: 'Number of parallel tests to run.'),
-    string(name: 'REPEAT_TESTS',
-           defaultValue: '1',
-           description: 'How many times to repeat each test.'),
-    string(name: 'ROBOTEST_VERSION',
-           defaultValue: 'stable-gce',
-           description: 'Robotest tag to use.'),
-    choice(choices: ["false", "true"].join("\n"),
-           defaultValue: 'false',
-           description: 'Whether to use preemptible VMs.',
-           name: 'GCE_PREEMPTIBLE'),
-    choice(choices: ["custom-4-8192", "custom-8-8192"].join("\n"),
-           defaultValue: 'custom-8-8192',
-           description: 'VM type to use.',
-           name: 'GCE_VM'),
-  ]),
-])
+// Define Robotest config that may be tweaked per job.
+// This is needed for the Jenkins GitHub Branch Source Plugin
+// which creases a unique Jenkins job for each pull request.
```

Comment:
**creates** a unique

Reply:
This mus be from before I turned on spellcheck in vim. :) I'll deal with this in a subsequent PR.  I'd like to get this merged to 6.3 to prevent further disruption.  I know @r0mant's got some other work headed into this branch.